ceph-releases/jewel/ubuntu/14.04/demo/entrypoint.sh

```diff
@@ -155,6 +155,12 @@ function bootstrap_osd {
     chown -R ceph. /var/lib/ceph/osd/${CLUSTER}-0
     ceph-osd ${CEPH_OPTS} -i 0 --mkfs --setuser ceph --setgroup ceph
     ceph ${CEPH_OPTS} auth get-or-create osd.0 osd 'allow *' mon 'allow profile osd' -o /var/lib/ceph/osd/${CLUSTER}-0/keyring
+    if [ $(hostname -s) == 'default' ]
```

Comment:
would you mind sticking with our convention? this will basically be:

 `if [ $(hostname -s) == 'default' ]; then``

Thanks!